src/library/library.h

```diff
@@ -95,6 +95,7 @@ class Library : public QObject {
   signals:
     void showTrackModel(QAbstractItemModel* model);
     void switchToView(const QString& view);
+    void updateTrackModel();
```

Comment:
I'm not sure about this naming if this isn't emitted every time the track model is updated.  Is there a more specific name we can give this, or are there more places where we should be emitting?